SETUP.md

```diff
@@ -3,45 +3,36 @@
 
 This document describes how to set up your workstation to develop for Code.org.
 
-You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in a VM/WSL). Setup for Windows is more complicated and relatively few developers use it. Make sure you follow the instructions for your platform in the subsections below.
+You can do Code.org development using macOS, Ubuntu, or Windows (running Ubuntu in a VM/WSL). Setup for Windows is more complicated and relatively few developers use it. Make sure you follow the instructions for your platform in the subsections below.
 
 ## Overview
 
 1. Request and Configure AWS access (code.org staff) or configure local secrets (open source contributors). See [Configure AWS Access or Secrets](#configure-aws-access-or-secrets) below. This step is not required until rake is first run below, but staff may wish to submit the request first so its ready when rake is.
-    <details> 
-      <summary>Troubleshoot: wrong version of ruby</summary>
-      If you run into issues with your ruby version during this step, you may need to complete the OS-specific  prerequisites below before proceeding. 
-      </details>
+
+1. Clone the repo:
+    - Via SSH (simpler): `git clone git@github.com:code-dot-org/code-dot-org.git`
+    - OR via HTTP (faster): `git clone https://github.com/code-dot-org/code-dot-org.git`.
+        - Although faster than SSH, this option requires you to reauthenticate every time you want to update. You will therefore probably want to switch to SSH after the initial clone with: `git remote set-url origin git@github.com:code-dot-org/code-dot-org.git`
+
+1. `cd code-dot-org`
 
 1. Install OS-specific prerequisites
-   - See the appropriate section below: [macOS](#macos), [Ubuntu](#ubuntu-2004), [Windows](#windows)
-   - *Important*: When done, check for correct versions of these dependencies:
+    - See the appropriate section below: [macOS](#macos), [Ubuntu](#ubuntu-2004), [Windows](#windows)
+    - *Important*: When done, check for correct versions of these dependencies:
 
      ```sh
      ruby --version  # --> ruby 3.0.5
      node --version  # --> v18.16.0
      ```
 
-1. Clone the repo, which also may take a while.
-    - Note you should have `git lfs --version` >= 3.0 installed prior to cloning, see OS-specific install steps referenced above.
-    - The simplest option is to clone via SSH with: `git clone git@github.com:code-dot-org/code-dot-org.git`
-    - The fastest option is to clone via HTTP with: `git clone https://github.com/code-dot-org/code-dot-org.git`. Although faster than SSH, this option requires you to reauthenticate every time you want to update. You will therefore probably want to switch to SSH after the initial clone with `git remote set-url origin git@github.com:code-dot-org/code-dot-org.git`
```

Comment:
We now suggest cloning the repo before doing os-specific steps. While this requires adding a `git lfs pull` below, it means that `nvm install` and `rbenv install` actually work in the OS specific sections. i.e.: this ordering is actually correct!

Reply:
should we list `git lfs` in the dependency check instructions above?

Reply:
Added!